client/my-sites/importer/logos/medium.jsx

```diff
@@ -5,10 +5,10 @@
  */
 import React from 'react';
 
-export default () => (
+export default ( { className = 'medium-logo social-logo importer__service-icon' } ) => (
 	<svg
 		/*eslint-disable wpcalypso/jsx-classname-namespace */
```

Comment:
Do we still need these `eslint-disable`s? If so, should replace with `eslint-disable-next-line` to tidy things up a bit.

Reply:
The linter complains when I try to remove this one, so I've changed it to a single line comment.